genesys:
  adma_ros2_driver:
    ros__parameters:
      destination_ip: '192.168.88.255'
      destination_port: 11021
      use_performance_check: False
      protocol_version: 'v3.3.4'
      frame_ids:
        navsatfix: 'gnss_link'
        imu: 'imu_link'
        adma: 'adma'
        adma_status: 'adma_status'
        raw_data: 'data_raw'
        odometry_pose_id: 'adma' # header.frame_id (basicly could be a "name" of desired POI)
        odometry_twist_id: 'odometry' # odometry.child_frame_id
      odometry_yaw_offset: 0.0 # in degrees
